### ISSUE TITLE
librenms: lnms script fails due to file ownership

### ISSUE THREAD

**Issue by sgrimee** [NixOS/nixpkgs#322344]
### Describe the bug
The `lnms` utility from the `librenms` package does not start because it checks that its owner is the _librenms_ user and not _root_.
However, nix sets the ownership of all package files in the store to _root_ so the script always fails.

### Steps To Reproduce
1. install the `librenms` 23.9.1 package 
2. as the _librenms_ user, in the package directory, try to run the `lnms` script
3. you'll get the error message: `Error: lnms file is owned by root, it should be owned and ran by a non-privileged user.`

### Expected behavior
The `lnms` script should launch when run by the _librenms_ user.

### Additional context
This was a small annoyance in the initial package version 23.9.1 but using the librenms-artisan wrapper was a reasonable workaround.
However, since version 23.10.0, [the poller.php script has been deprecated](https://github.com/librenms/librenms/pull/15370) and the `lnms` script is used instead. This is a showstopper as it causes polling to fail for the entire application, making it unusable.

The check is done in function `runningUser` in the `app/Checks.php` file. It checks for the owner of the file, not the running user.

Removing that test fixes the issue and the `lnms` script works.
On _librenms_ versions > 23.10.0, this also restores polling functionality.

### Notify maintainers
@NetaliDev @r-ryantm @johannwagner 

### Metadata

```console
[user@system:~]$ nix-shell -p nix-info --run "nix-info -m"
 - system: `"x86_64-linux"`
 - host os: `Linux 6.6.32, NixOS, 24.11 (Vicuna), 24.11.20240624.dirty`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.18.3`
 - channels(root): `"nixos"`
 - nixpkgs: `/nix/store/w1sm9m0dv0g95njr6vz8bg4ibxh2kwm2-source`
```

---

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

**Comment by johannwagner** [comment]
I will additionally ping @yu-re-ka 🇩🇰

**Comment by sgrimee** [comment]
I have just pushed a proposed fix in my PR #315390.
It removes the check and adjusts the existing lnms wrapper in the `librenms` module to use the correct user.

**Comment by sklemer1** [comment]
As discussed with @yu-re-ka : Tried the fix today -- works for us; all fine.
